.github/workflows/release.yml

```diff
@@ -33,34 +44,86 @@ jobs:
   linux:
     strategy:
       matrix:
-        openssl:
-          - 1_1_0
-          - 1_0_1
-    name: Build - OpenSSL ${{ matrix.openssl }}
+        arch: [x86_64, arm64]
+        openssl: [1_0_1, 1_1_1]
+        os: [linux]
+        include:
+        - arch: x86_64
+          dockerfile: Dockerfile.centos6
+          # Docker's BuildX platform for x86_64 is `amd64`. But `uname -m` on linux returns x86_64.
+          # So we'll officially designate this binary as `x86_64` but map it here to `amd64` so that
+          # we build it on the correct docker arch
+          dockerArch: amd64
+        - arch: arm64
+          dockerfile: Dockerfile.centos8
+          dockerArch: arm64
+
+    name: Build - OpenSSL ${{ matrix.openssl }} - Arch ${{ matrix.arch }}
     runs-on: ubuntu-latest
     steps:
+      
       - name: Check out code
         uses: actions/checkout@v2
-      - name: Check out OpenSSL
-        uses: actions/checkout@v2
+
+      # Not stricly speaking necessary for x86 builds, but the QEMU arm64 builds take hours
```

Comment:
Given that the arm64 builds take so long, is there a way we can lock them in the same way as the final `Publish release` job, so that they only run when we're actually publishing a new version? My worry is that currently this build is part of the CI check for PRs and I don't want to run into a spot where every PR is gated on a build that takes ~2 hours.